prow/plugins/config.go

```diff
@@ -437,6 +438,34 @@ type ProjectRepoConfig struct {
 	ProjectColumnMap map[string]string `json:"repo_default_column_map,omitempty"`
 }
 
+// ProjectManager represents the config for the ProjectManager plugin, holding top
```

Comment:
Godoc seems stale already

Reply:
Sorry?, The Good explain what a project manager does. It is a automated way to enable PR's to be added to a project if the project is configured.

Reply:
The struct already does not hold a list of projects as the GoDoc claims

Reply:
modified the statement, please see.